### PR TITLE
#2674 fixing several resource handling misses

### PR DESCRIPTION
**Pull request by svendiedrichsen**
Fixes some of the resource handling issues found by SonarQube in #2674

### CODE REVIEW

**Comment by Cousjava** [comment]
Jenkins test please

**Comment by svendiedrichsen** [comment]
@lprimak Sorry, I have found some time to do some more fixes.

**Comment by mulderbaba** [comment]
Jenkins test please

File: appserver/extras/payara-micro/payara-micro-boot/src/main/java/fish/payara/micro/boot/PayaraMicroLauncher.java [hunk 1]
```diff
@@ -65,46 +65,45 @@
...
      * Boot method via Micro.getInstance()
-     * @return
-     * @throws InstantiationException
-     * @throws IllegalAccessException
-     * @throws ClassNotFoundException
-     * @throws Exception 
+     * @return PayaraMicroBoot instance
+     * @throws Exception error
      */
-    public static PayaraMicroBoot getBootClass() throws InstantiationException, IllegalAccessException, ClassNotFoundException, Exception {
```
[the remark this answers is not in the document]
Comment (svendiedrichsen) [reply appserver/extras/payara-micro/payara-micro-boot/src/main/java/fish/payara/micro/boot/PayaraMicroLauncher.java]: Sure. But I'd argue that this Javadoc does not do much to clarify anything.

**Comment by svendiedrichsen** [comment]
@mulderbaba Do you want me to bring the whitespaces back?

**Comment by lprimak** [comment]
GitHub diff screen has a setting "hide whitespace changes" that was very helpful in this case

**Comment by lprimak** [comment]
jenkins test

**Comment by svendiedrichsen** [comment]
@mulderbaba As @lprimak noted you can make Github show the diff without spaces.

**Pull request merged**